### PR TITLE
Update Next config in SSR docs

### PR DESCRIPTION
**Pull request by reidbarber**
We were missing the avatar package after the latest release . Also updated the 13.1 and older instructions to match our latest. 

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### CODE REVIEW

**Review by LFDanLu — DISMISSED** [review]

File: packages/dev/docs/pages/react-spectrum/ssr.mdx [hunk 1]
```diff
@@ -128,17 +129,20 @@ With this installed, add the following to your `next.config.js` file.
...
   "@react-spectrum/actiongroup",
-  "@react-spectrum/breadcrumbs",
+  "@react-spectrum/avatar",
   "@react-spectrum/badge",
+  "@react-spectrum/breadcrumbs",
   "@react-spectrum/button",
   "@react-spectrum/buttongroup",
   "@react-spectrum/calendar",
   "@react-spectrum/checkbox",
+  "@react-spectrum/color",
```
Comment (snowystinger) [inline packages/dev/docs/pages/react-spectrum/ssr.mdx:139]: i don't see color in the monorepo?

Comment (reidbarber) [reply packages/dev/docs/pages/react-spectrum/ssr.mdx:139]: Good catch. Interesting that the config still works though. I'll remove this.

Comment (reidbarber) [reply packages/dev/docs/pages/react-spectrum/ssr.mdx]: As a sanity check, I confirmed that it still works without it.

Comment (snowystinger) [reply packages/dev/docs/pages/react-spectrum/ssr.mdx]: I think it just tells what nextjs to do if it encounters those modules? so if you're using any of our beta/alpha/rc etc packages, you'd need to add them here

**Review by LFDanLu — APPROVED** [review]

**Pull request merged**